src/language-css/parser-postcss.js

```diff
@@ -200,12 +200,6 @@ function parseValue(value) {
   return addTypePrefix(parsedResult, "value-");
 }
 
-function parseMediaQuery(value) {
-  const mediaParser = require("postcss-media-query-parser").default;
```

Comment:
Can we remove from package.json as well?

Reply:
@lydell yep :+1: